src/supermarket/db/migrate/20160914215547_move_quality_metric_results.rb

```diff
@@ -0,0 +1,26 @@
+class MoveQualityMetricResults < ActiveRecord::Migration
+  def change
+    foodcritic_qm = QualityMetric.create!(name: 'Foodcritic')
+    collab_num_qm = QualityMetric.create!(name: 'Collaborator Number')
```

Comment:
I wonder if we need something to create these in `seeds.rb`, too, for systems built up with `db:schema:load` (I can think of the test environments, [in travis](https://github.com/chef/supermarket/blob/be7bf10424daa999bee8e5dfac4e87d14b786abe/src/supermarket/lib/tasks/travis.rake#L7) and [in automate](https://github.com/chef/supermarket/blob/be7bf10424daa999bee8e5dfac4e87d14b786abe/.delivery/build_cookbook/recipes/unit.rb#L56)).